buildSrc/src/main/kotlin/Versions.kt

```diff
@@ -0,0 +1,43 @@
+import org.gradle.api.JavaVersion
+
+object Versions {
+
+    const val project = "0.0.1"
+
+    object Compile {
+        const val kotlin = "1.5.0"
+        val sourceCompatibility = JavaVersion.VERSION_1_8
+        val targetCompatibility = JavaVersion.VERSION_1_8
+        val jvmTarget = targetCompatibility.name.removePrefix("VERSION_").replace('_', '.')
+    }
+
+    object Plugins {
+        const val kethabi = "0.2.0"
+        const val allOpen = Compile.kotlin
+        const val ktlint = "10.0.0"
+        const val detekt = "1.17.1"
+        const val testSets = "4.0.0"
+        const val testLogger = "3.0.0"
+        const val micronaut = "1.5.0"
+        const val shadowJar = "6.1.0"
+    }
+
+    object Tools {
+        const val ktlint = "0.41.0"
+        const val micronaut = "2.5.4"
+        const val jacoco = "0.8.7"
+    }
+
+    object Dependencies {
+        const val okHttp = "4.9.1"
+        const val graalSdk = "21.0.0.2"
+        const val web3jCore = "5.0.0"
```

Comment:
https://github.com/web3j/web3j. This version is not listed as released on their github page. Is this is a stable version?

Reply:
From the version number it should be regarded as stable, as far as the versioning standards go. It could be that they did not update the documentation yet. In any case, I did not run into any trouble using this newer version and it is available on Maven central: https://mvnrepository.com/artifact/org.web3j/core